installer/PowerToysSetup/Product.wxs

```diff
@@ -206,10 +206,9 @@
         </Directory>
       </Directory>
       <Directory Id="ProgramMenuFolder">
-        <Directory Id="ApplicationProgramsFolder" Name="PowerToys"/>
-      </Directory>
       <Directory Id="DesktopFolder" Name="Desktop"/>
     </Directory>
+   </Directory>
```

Comment:
we need to remove this line

Reply:
I am getting an error "Tag was not closed" when that line was not added. 
That was why I added the extra line to end the directory tag

Reply:
ah, alright

Reply:
that is because line 209 and 210 were removed.  208 should be a /> or just removed

Reply:
I have tested both cases, Removing line 208 would result in an "Unresolved reference to symbol error" because "ProgramMenuFolder" is being used in the shortcut definition on line 225.

changing line 208 to a /> returns the same error

Reply:
something here isn't right due to nesting.  With how the code is now  i don't think desktop folder is supposed to be nested in to programmenufolder.

this is the new nesting
```
    <Directory Id="ProgramMenuFolder">
      <Directory Id="DesktopFolder" Name="Desktop"/>
    </Directory>
```

before it was this
```
      <Directory Id="ProgramMenuFolder">
        <Directory Id="ApplicationProgramsFolder" Name="PowerToys"/>
      </Directory>
      <Directory Id="DesktopFolder" Name="Desktop"/>
```